ydb/core/scheme/scheme_tablecell.h

```diff
@@ -120,6 +120,26 @@ struct TCell {
     const char* InlineData() const                  { Y_VERIFY_DEBUG(!IsInline_); return Ptr; }
     const char* Data() const                        { Y_VERIFY_DEBUG(!IsInline_); return Ptr; }
 #endif
+
+    void CopyData(char * dst) const {
```

Comment:
Name like `CopyData` lacks direction, e.g. could it maybe copy data into the cell? Maybe name it `CopyDataTo(dst)`?

Reply:
Updated.